yahtzee.py

```diff
@@ -0,0 +1,28 @@
+#import packages
+from random import randint
+import numpy as np
+import pygame as pg
+
+'''  Check classes tomorrow for DiceSet?
+class Dice:
+    def __init__(self, value):
+        self.value = value        
+ '''      
+
+
+num_throws = 3
+num_dice = 5
+dice = np.zeros([num_dice])
+for i in range(0, num_throws):
+    if i == 0:
+        throw_again_idx = range(0, len(dice))
+    else:
+        throw_again_idx = np.array([int(i) - 1 for i in input().split()])
```

Comment:
Maybe add a comment here to explain what you're doing. How I understand it you assign a different number of throws depending on if it's the player's first turn or not?

Reply:
Yeah, I should definitely make more comments haha. Forgot to add them in my enthusiasm when the code was finally working